cylc/flow/exceptions.py

```diff
@@ -435,13 +435,30 @@ def __str__(self):
         return f'Unable to find valid host for {self.platform_name}'
 
 
-class NoPlatformsError(PlatformLookupError):
-    """None of the platforms of a given group were reachable."""
-    def __init__(self, platform_group):
-        self.platform_group = platform_group
+class NoPlatformsError(CylcError):
```

Comment:
This regression was the reason for some of the failing tests
```suggestion
class NoPlatformsError(PlatformLookupError):
```